src/workerd/api/streams/queue.h

```diff
@@ -263,18 +270,8 @@ class QueueImpl final {
   struct Closed {};
   using Errored = jsg::Value;
 
-  struct ConsumerRef {
-    ConsumerImpl* ref;
-    bool operator==(ConsumerRef& other) const {
-      return hashCode() == other.hashCode();
-    }
-    auto hashCode() const {
-      return kj::hashCode(ref);
-    }
-  };
-
   struct Ready final: public State {
-    kj::HashSet<ConsumerRef> consumers;
+    std::set<ConsumerImpl*> consumers;
```

Comment:
I'm curious how high-traffic is this data structure? I ask because switching from `kj::HashSet` to `std::set` could potentially impact performance, though it's probably unlikely.